fcl/dunefdvd/gen/solar/prodmarley_wbkg_dunevd10kt_1x8x14_3view_30deg.fcl

```diff
@@ -0,0 +1,26 @@
+#include "marley_dune.fcl"
```

Comment:
"prodmarley_wbkg_dunevd10kt_1x8x14_3view_30deg.fcl" uses a different naming scheme (wbkg) to the other fcls.  Could this be changed?

Reply:
Renamed